### PR TITLE
Optis and cleanups

### PR DESCRIPTION
**Pull request by phlax**
as in title

### CODE REVIEW

File: pootle/apps/pootle_store/fields.py [hunk 1]
```diff
@@ -152,65 +135,23 @@ def realpath(self):
...
+
+        fileclass = self.instance.syncer.file_class
+        classes = {
+            str(self.instance.filetype.extension): fileclass,
+            str(self.instance.filetype.template_extension): fileclass}
+        return factory.getobject(
+            self.path,
+            ignore=self.field.ignore,
+            classes=classes)
         return self._store_tuple.store
```
Comment (ta2-1) [inline pootle/apps/pootle_store/fields.py]: return after return

File: pootle/apps/pootle_store/fields.py [hunk 2]
```diff
@@ -152,65 +135,23 @@ def realpath(self):
         else:
             return self._store_tuple.realpath
```
Comment (unho) [inline pootle/apps/pootle_store/fields.py]: `_store_tuple` can be dropped too, now that it is not populated. `realpath` would need merging with `_get_realpath`.

File: pootle/apps/pootle_store/fields.py (hunk 1 above)
Comment (unho) [inline pootle/apps/pootle_store/fields.py]: This line is never reached.

**Comment by unho** [comment]
lgtm

**Pull request merged**